### PR TITLE
Change pause button position for carousels

### PR DESCRIPTION
**Pull request by MewenLeHo**
### Related issues

[#1578](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1578) 

### Description

Change the position of the pause button for better accessibility compliance.

### Motivation & Context

Users of assistives technologies need to have the choice to pause or play a carousel before receiving the information.

### Types of change

- Refactoring (non-breaking change)

### Live previews

* https://deploy-preview-1593--boosted.netlify.app/docs/5.2/components/carousel/#with-pauseplay-button

### Checklist

- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)
- [x] My code follows the code style of the project _(using `npm run lint`)_
- [x] My change introduces changes to the documentation
- [x] I have updated the documentation accordingly
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed

### CODE REVIEW

**Comment by MewenLeHo** [comment]
Please note that this change was made to provide a preview for accessibility experts and designers.
We can make this change for prevent accessibility issues while waiting for the next step: ading the play/pause button next to the indicators dots.

Possible evolution: add a callout in documentation to explain why it is important to keep the play/pause button in first position?

**Comment by julien-deramond** [comment]
> Possible evolution: add a callout in documentation to explain why it is important to keep the play/pause button in first position?

Yep I think this would be interesting to explain it in an info callout for the users. Otherwise, they will maybe switch the positions.

**Comment by julien-deramond** [comment]
This PR will need an explanation in the migration guide.

File: site/content/docs/5.2/components/carousel.md [hunk 1]
```diff
@@ -103,11 +103,18 @@ Carousel progress indicator is paused under multiple conditions:
 {{< added-in "5.2.0" >}}
 
 Adding a pause and play button is recommended to setup an accessible carousel.
-This `button` must immediately follow your carousel and have the custom `data-bs-control="play-button"` attribute. In addition, it must also have a `data-bs-target` attribute that matches the `id` of the `.carousel` element.
+This `button` must have the custom `data-bs-control="play-button"` attribute. In addition, it must also have a `data-bs-target` attribute that matches the `id` of the `.carousel` element.
 
 'Play' and 'Pause' texts can be changed by modifying `data-bs-play-text` and `data-bs-pause-text` custom attributes.
 
+{{< callout warning >}}
+For accessibility purposes, the pause/play button must be placed before the carousel.
```
Comment (louismaximepiton) [inline site/content/docs/5.2/components/carousel.md:111]: `should` instead of `must` might be better here.

**Review by louismaximepiton — COMMENTED** [review]
Waiting for the migration guide to updated before approving.

**Pull request closed without merging**